### PR TITLE
Security rules

### PR DESCRIPTION
**Pull request by wbellingrath-jnpr**

### CODE REVIEW

**Comment by gvarao** [comment]
Jenkins please retry a build

File: community_supplied/Security/get-dev-key-state.rule [hunk 1]
```diff
@@ -0,0 +1,89 @@
+/*
+ * This rule checks if development keys have been left on the system.
+ * 
+ * If development keys were present, this rule will report the results
+ * of the attempted revocation as Successful or a Failure.
+ * This rule relies on a file present in 19.3R1 and further.
+ */
+healthbot {
+    topic security {
+        rule get-dev-key-status {
```
Comment (gvarao) [inline community_supplied/Security/get-dev-key-state.rule]: Thanks for contributing HB rules.
If possible, Please add rule "description" and "synopsis"

File: community_supplied/Security/test-secure-boot-status.rule [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+ *
+ * This rule relies on a file present in 19.3R1 and beyond.
+ * This rule is only applicible to devices where secureboot is capable.
+ * If this rule is run on a device that is incapable of secureboot,
+ * the result will be "secureboot status is unknonw" and the rule should be
+ * removed.
+ */
+healthbot {
+    topic security {
+        rule check-secureboot-status {
```
Comment (gvarao) [inline community_supplied/Security/test-secure-boot-status.rule]: If possible, please add rule description and synopsis. So it provides information about rule functionality. BTW comments provide information about rule but comments disappear after loading rule in HB MGD. Hence requesting you to add description and synopsis

**Comment by prashant3167** [comment]
jenkins please retry a build

**Pull request merged**